src/coreclr/jit/gentree.h

```diff
@@ -7303,6 +7302,10 @@ struct GenTreeBlk : public GenTreeIndir
     enum
     {
         BlkOpKindInvalid,
+        BlkOpKindCpObjUnroll,
+#ifdef TARGET_XARCH
+        BlkOpKindCpObjRepInstr,
+#endif
```

Comment:
These are missing handling in `gtDispTree`, can you submit a fix? (They currently hit an `unreached()`)

Reply:
Working on it.